lib/xapi-stdext-date/date.mli

```diff
@@ -24,7 +24,8 @@ val of_float : float -> iso8601
 (** Convert date/time to a float value: the number of seconds since 00:00:00 UTC, 1 Jan 1970. *)
 val to_float : iso8601 -> float
 
-(** Convert date/time to an ISO 8601 formatted string. *)
+(** Convert date/time to an ISO 8601 formatted string.
+  * Does not accept a timezone annotated datetime *)
```

Comment:
Not sure why this line got added, did it make sense before switching to the latest approach?

Reply:
i meant to add this to `of_string`